app/services/add_sandbox_template.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require_dependency "#{Rails.root}/lib/wiki_api"
+require_dependency "#{Rails.root}/lib/wiki_edits"
+
+class AddSandboxTemplate
+  def initialize(home_wiki:, sandbox:, sandbox_template:, current_user:)
+    @current_user = current_user
+    @sandbox = sandbox
+    @default_template = '{{user sandbox}}'
+    @sandbox_template = sandbox_template
+    @wiki_editor = WikiEdits.new(home_wiki)
+    @wiki_api = WikiApi.new(home_wiki)
```

Comment:
If this is only used in `initialize`, it should be a local variable rather than an instance variable. In general, the fewer instance variables a class has, the easier it is to understand how it works and what the variables are for.

Reply:
Done!